dockerbuild/README.md

```diff
@@ -66,4 +66,4 @@ It is also possiblet to run ModelDB in Docker without Docker Compose.
     docker start mongo backend frontend
     ```
 
-    Shortly after running this command, ModelDB should be reachable at [http://localhost/](http://localhost/).
+    Shortly after running this command, ModelDB frontend will be reachable at [http://localhost/3000](http://localhost/3000) and you will be able to log data into ModelDB via ModelDB clients.
```

Comment:
Should we also mention that the ModelDB server to log data into will be at localhost:6543?